packages/compass-shell/src/components/compass-shell/compass-shell.jsx

```diff
@@ -21,10 +28,11 @@ const compassShellContainerStyles = css({
   display: 'none',
   overflow: 'auto',
   borderTop: `1px solid ${palette.gray.dark2}`,
-  '*::-webkit-scrollbar-thumb': {
-    background: rgba(palette.gray.light1, 0.5),
-  }
-});
+  // TODO(COMPASS-6206): After this package is on the new config
+  // we can start using a darkMode ThemeProvider in this component
+  // so we can do things like `useScrollbars()` without explicity
+  // using these scrollbar styles.
```

Comment:
What's preventing you from using it on the current config? I am not sure how is this related 🤔

Reply:
I didn't want to convert it to a functional component without being able to also typescript it. Not totally a requirement, but figured it might make the swap a bit smoother.